libs/application/templates/driving-license/src/lib/drivingLicenseTemplate.ts

```diff
@@ -70,17 +103,32 @@ const template: ApplicationTemplate<
           ],
         },
         on: {
+          '*': { target: States.PAYMENT_PENDING },
           [DefaultEvents.SUBMIT]: { target: States.DONE },
```

Comment:
What is "*" for? Should the user be able to submit the form?

Reply:
Will be discussed.